src/document.h

```diff
@@ -1716,6 +1719,19 @@ struct Document {
         return nullptr;
     }
 
+    const wxChar *FilterMatchNext(wxDC &dc) {

```

Comment:
Please inline this function

Reply:
Thanks for your review! I revised the commit.